src/svcutilcore/src/Microsoft/Tools/ServiceModel/SvcUtil/ExportModule.cs

```diff
@@ -48,24 +45,26 @@ internal TypeLoadErrorEventHandler ContractLoadErrorCallback
                 set { _contractLoadErrorCallback = value; }
             }
 
-            internal IEnumerable<ContractDescription> GetContracts()
+            internal IEnumerable<Object> GetContracts()
             {
                 foreach (Type type in _types)
                 {
                     if (!_isTypeExcluded(type) && IsContractType(type))
                     {
-                        ContractDescription contract = LoadContract(type);
+                        var contract = LoadContract(type);
                         if (contract != null)
                             yield return contract;
                     }
                 }
             }
 
-            private ContractDescription LoadContract(Type type)
+            private object LoadContract(Type type)
             {
                 try
                 {
-                    ContractDescription description = ContractDescription.GetContract(type);
+                    Type contractDescription = Tool.SMAssembly.GetType("System.ServiceModel.Description.ContractDescription");
+                    MethodInfo GetContract = contractDescription.GetMethod("GetContract", new Type[] { typeof(Type) });
+                    object description = GetContract.Invoke(null, new object[] { type });
```

Comment:
Use a different overload which accepts BindingFlags and be more specific. The method is public and static